proto/proto/block_producer.proto

```diff
@@ -29,10 +25,10 @@ service Api {
     //
     // Currently only a single active subscription is supported. Subscription requests will cancel the active
     // subscription, if any.
-    rpc MempoolSubscription(MempoolSubscriptionRequest) returns (stream MempoolEvent) {}
+    rpc MempoolSubscription(MempoolSubscription) returns (stream MempoolEvent) {}
 }
 
-message MempoolSubscriptionRequest {
+message MempoolSubscription {
```

Comment:
I would keep this as `MempoolSubscriptionRequest`. Using `MempoolSubscription` makes it sound too much like a message containing mempool subscription.

Also, let's add `MEMPOOL SUBSCRIPTION` section separator above this line.